docs/sources/clients/promtail/configuration.md

```diff
@@ -1016,6 +1019,57 @@ When Promtail receives GCP logs, various internal labels are made available for
 - `__gcp_resource_type`
 - `__gcp_resource_labels_<NAME>`
 
+### Azure Event Hub
```

Comment:
Is it Azure Event Hub (singular) or Azure Event Hubs (plural)?  I'm seeing both forms in the docs update when it should be consistent if it's a product name.

Reply:
@JStickler nice catch 👍 

I did check Azure [catalog](https://azure.microsoft.com/en-us/products/), and the service name is `Event Hubs` (plural).

I will update the code and docs to use plural everywhere.

Thank you for pointing it out 👍